tests/resolvers/software-info.spec.ts

```diff
@@ -0,0 +1,78 @@
+// For clarity in this example we included our typeDefs and resolvers above our test,
+
+import { version } from '../../package.json';
+import ContextProvider from '../testContext';
+
+describe('Query should return Software info', () => {
+  it('All data should be returned', async () => {
+    const response =
+      await ContextProvider.Instance.apolloTestServer.executeOperation({
+        query: 'query { softwareInfo { title status version } }',
+      });
+
+    expect(response).toBeDefined();
+    expect(response.errors).toBeUndefined();
+    expect(response.data).toBeDefined();
+    const data = response.data as any;
+    expect(data.softwareInfo).toBeDefined();
+    expect(data.softwareInfo.length).toBeGreaterThan(0);
+    const softwareInfo = data.softwareInfo[0];
```

Comment:
We can remove usage of `any` and use [optional chaining operator (`?.`)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining).
```suggestion
    const data = response.data;
    expect(data?.softwareInfo).toBeDefined();
    expect(data?.softwareInfo.length).toBeGreaterThan(0);
    const softwareInfo = data?.softwareInfo[0];
```